lib/index.js

```diff
@@ -152,7 +172,39 @@ export function configureUnifiedLanguageServer(
    * @param {boolean} alwaysStringify
    * @returns {Promise<VFile[]>}
    */
-  function processDocuments(textDocuments, alwaysStringify = false) {
+  async function processDocuments(textDocuments, alwaysStringify = false) {
+    /** @type {EngineOptions['processor']} */
+    let processor
+
+    try {
+      // @ts-expect-error: assume we load a unified processor.
+      processor = await loadPlugin(processorName, {
+        cwd: process.cwd(),
+        key: processorSpecifier
+      })
+    } catch (error) {
+      const exception = /** @type {NodeJS.ErrnoException} */ (error)
+
+      // Pass other funky errors through.
+      /* c8 ignore next 3 */
+      if (exception.code !== 'ERR_MODULE_NOT_FOUND') {
+        throw error
+      }
+
+      if (!defaultProcessor) {
+        connection.window.showInformationMessage(
+          'Cannot turn on language server without `' +
+            processorName +
+            '` locally. Run `npm install ' +
+            processorName +
+            '` to enable it'
+        )
+        return []
+      }
```

Comment:
This information may be valuable if users report new issues.

```js
else {
  window.console.log('Failed to import `' + processorName + '`. Falling back to defaultProcessor')
}
```

Reply:
Do you mean `connection.console.log`?
- Why print to the console, and not a message?
- How does `window/logMessage` end up in an issue?

Reply:
> Do you mean `connection.console.log`?

Yes :facepalm: 

> * Why print to the console, and not a message?
> * How does `window/logMessage` end up in an issue?

I specifically mean in an `else` clause.

Visual Studio Code can display these messags in the output console. `vim-lsp` can write a log file if the following is added to `.vimrc`:

```viml
let g:lsp_log_file = expand('~/.cache/vim-lsp.log')
```